### PR TITLE
Add test script for the container

### PR DESCRIPTION
**Pull request by mnagy**

### CODE REVIEW

File: 5.5/test/run [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+		fi
+		attempt=$(( $attempt + 1 ))
+		sleep $sleep_time
+	done
+	echo "  Giving up: Failed to connect"
+	return 1
+}
+
+test_mysql() {
+	echo "  Testing table creation"
```
Comment (soltysh) [inline 5.5/test/run]: You're actually doing create + insert + select + drop, so maybe some other comment than table creation?

**Comment by soltysh** [comment]
Generally OK, but I think you're missing something like [hack/test.sh](https://github.com/openshift/sti-nodejs/blob/master/hack/test.sh) in this PR, because who is going to build the `openshift/mysql-55-centos7-candidate` for you?

**Comment by mnagy** [comment]
Agreed. I think I'll try add the -candidate building logic into the Makefile. I don't want another shell script duplicating the functionality.

**Comment by mnagy** [comment]
Added test target to Makefile. PTAL.
Note that I'd prefer not to squash this, but keep it as a separate commit.

**Comment by soltysh** [comment]
LGTM.

**Comment by soltysh** [comment]
LGTM :+1:

**Pull request merged**